layouts/_default/baseof.html

```diff
@@ -73,6 +73,17 @@ <h1 class="page-title">{{ .Title }}</h1>
         });
     </script>
 
+    <script>
+      function copyPermalink(link) {
+          // Copy the article link to the clipboard
+          let sanitizedLink = link;
+          while (sanitizedLink[0] === '/') {
+              sanitizedLink = sanitizedLink.substring(1);
+          }
+          navigator.clipboard.writeText(sanitizedLink);
```

Comment:
Last change, I promise 😂 
```suggestion
          navigator.clipboard.writeText(RegExp('^/*(.*)').exec(link)[1]);
```

Reply:
not a big fan of loops 💁🏽

Reply:
Thanks haha. This looks way better.